### PR TITLE
Add html files for showing benchmark log data

### PR DESCRIPTION
**Pull request by cgx027**
The footprint benchmark html files will be copied to benchmark log data folder, where the log data will be shown as a html page using codes in this PR. 

@RackHD/corecommitters @WangWinson @rolandpoulter @wuy1

### CODE REVIEW

File: footprint-benchmark/html/report/config.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
+var configuration =
```
Comment (WangWinson) [inline footprint-benchmark/html/report/config.js]: Add `// jshint ignore:line` maybe ?

File: footprint-benchmark/html/report/detail.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                    tr.appendChild(td3);
+                    tr.appendChild(td4);
+                    
+                    $('#measure_body').append(tr);
+                })
+            }
+            
+        });
+
+    showGraph(measure_data,
```
Comment (WangWinson) [inline footprint-benchmark/html/report/detail.html]: missing indent ?

File: footprint-benchmark/html/report/detail.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        script_element.setAttribute('type', 'text/javascript');
+        script_element.setAttribute('src', script_path);
+        node_head.append(script_element); //synchronize load
+    }
+
+    function getConfiguration(configuration, measure, key){
+        var value;
+        $.each(configuration, function(group, measure_obj){
+            if(measure_obj.hasOwnProperty(measure)){
+                value = measure_obj[measure][key];
```
Comment (WangWinson) [inline footprint-benchmark/html/report/detail.html]: why not `return value` directly?

File: footprint-benchmark/html/report/detail.html [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+                    },
+                    strokeWidth: 1,
+                    customBars: false,
+                    logscale: false
+                });
+    }
+
+    function getQueryStringParams(param){
+        var page_url = window.location.search.substring(1);
+        var page_vars = page_url.split('&');
```
Comment (WangWinson) [inline footprint-benchmark/html/report/detail.html]: may need parse url first ? `decodeURIComponent()`

File: footprint-benchmark/html/report/summary.html [hunk 5]
```diff
@@ -0,0 +1,639 @@
...
+    <script type="text/javascript" src="../data/atop_summary.js"></script>
+    <script type="text/javascript" src="../data/db_disk_summary.js"></script>
+    <script type="text/javascript" src="../data/db_doc_summary.js"></script>
+    <script type="text/javascript" src="../data/case_info.js"></script>
+    <script type="text/javascript" src="../data/process_list.js"></script>
+    <script type="text/javascript" src="../data/compare_list.js"></script>
+    <script type="text/javascript" src="config.js"></script>
+    <script type="text/javascript" id="compare_data" src=""></script>
+
+    <script type="text/javascript">
```
Comment (WangWinson) [inline footprint-benchmark/html/report/summary.html]: Best practice : move the script tag to the bottom of body

File: footprint-benchmark/html/report/detail.html [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+            }
+            
+        });
+
+    showGraph(measure_data,
+            getConfiguration(configuration, measure, 'Short Description'),
+            getConfiguration(configuration, measure, 'unit'));
+    });
+    
+    function zeropad(x) {
```
Comment (WangWinson) [inline footprint-benchmark/html/report/detail.html]: Best practice: if not a global function, move to closure, for example: in `$(document).ready`.

**Comment by WangWinson** [comment]
General comments: need refactor
- decouple view and data model, may leverage some template render.
- stick to unique code style convention
- use jQuery to avoid DOM operation directly

File: footprint-benchmark/html/report/detail.html (hunk 2 above)
Comment (cgx027) [inline footprint-benchmark/html/report/detail.html]: Goot catch indeed. Thanks.

File: footprint-benchmark/html/report/detail.html (hunk 3 above)
Comment (cgx027) [inline footprint-benchmark/html/report/detail.html]: Not bad. Suggestion taken

**Comment by cgx027** [comment]
Thanks @WangWinson, a lot of great suggestions. This PR is focused on submitting a working code for bootstrap. Your other refactor suggestions will be added int Pivotal Tracker as future actions.

**Comment by WangWinson** [comment]
:+1:  Understood @cgx027

**Comment by yyscamper** [comment]
+1

**Comment by cgx027** [comment]
Please ignore the HoundCI issue caused by minimized Jquery/Bootstrap libs.

**Pull request merged**